tests/resources/certificate/invalid_signature.signed.json

```diff
@@ -30,7 +30,7 @@
       "hash": "sha512",
       "encryption": "EdDSA"
     },
-    "value": "deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef",
+    "value": "839a2d43d22690338f4fb282e7bcc790e7352dc99e42dc603d43e51e4e09f2c709f1b9f1f4c90b446107440f8ab13345fbbd1d64337acf70b2777be9a522be0b",
```

Comment:
I modified it to a valid signature because the JS program blows up unexpectedly if the signature is not result of a valid computation. Changed a bit in public key to break verification.

Reply:
If so, then please edit it in other `invalid_signature` cases in descriptors to be sure that it would work with JS

Reply:
I just needed this one to use in the readme as example. I did not want that one to blow up with an exception as it might mean other problems. So only this one is fine.